src/test/scala/uk/gov/hmrc/perftests/dds/requests/DigitalDisclosureRequests.scala

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.dds.requests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import uk.gov.hmrc.perftests.dds.Configuration
+
+object DigitalDisclosureRequests extends Configuration {
+
+  val digitalDisclosureRoute = s"$digitalDisclosureUrl/digital-disclosure"
+  val notificationRoute      = s"$digitalDisclosureRoute/notification"
+
+  val navigateToStart: HttpRequestBuilder =
+    http("Navigate to /digital-disclosure")
+      .get(s"$digitalDisclosureRoute")
+      .check(status.is(200))
+
+  val navigateToLetterFromHmrc: HttpRequestBuilder =
+    http("Navigate to /letter-from-hmrc")
+      .get(s"$notificationRoute/letter-from-hmrc")
+      .check(status.is(200))
+      .check(saveCsrfToken)
+
+  def submitLetterFromHmrc(value: String): HttpRequestBuilder =
```

Comment:
This applies to all `GET `and `POST` requests throughout the project, but do **NOT** hard code the values you wish to send as part of the request as this makes it impossible to re-use the request.

For example here, pass the value into the function as a string like this, then in `journey-ABC` you can send `REF123`, and in `journey-XYZ` you can send `REF456` re-using the function for a near identical request on the same page across the journeys.